crates/async-sawtooth-sdk/src/error.rs

```diff
@@ -9,10 +11,13 @@ pub enum SawtoothCommunicationError {
     ZMQ(#[from] zmq::Error),
 
     #[error("Send error {0}")]
-    Send(#[from] SendError),
+    SendChannel(#[from] SendError),
+
+    #[error("Sawtooth receive error {0}")]
```

Comment:
```suggestion
    #[error("Sawtooth receive error: {0}")]
```
Still keen on if we can start including colons or somesuch into all these for easier reading.